charts/fleet-agent/templates/rbac.yaml

```diff
@@ -9,7 +9,10 @@ rules:
   - '*'
   verbs:
   - '*'
-
+- nonResourceURLs:
+  - "*"
+  verbs:
```

Comment:
That's a duplicate from lines 10/11 ?!

Reply:
The diff is pretty unreadable, but this is a new array element for `rules:`, so it comes out at

```
rules: [ 
  {"apiGroups": ['*'], "resources": ['*'], "verbs": ['*']}, 
  {"nonResourceURLs": ["*"], "verbs": ['*']}
 ],
```

Reply:
🙈 - github diffs again ...